README.md

```diff
@@ -58,6 +46,7 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
 ### On any platform with dub
 
 [![Dub version](https://img.shields.io/dub/v/mir.svg)](http://code.dlang.org/packages/mir)
+[![Dub downloads](https://img.shields.io/dub/dt/mir.svg)](http://code.dlang.org/packages/mir)
```

Comment:
This is only about dub and I move it to dub. A lot of shields in one place look weird